media/scss/base/font.scss

```diff
@@ -1,32 +1,96 @@
-* { font-family: 'Quattrocento Sans', sans-serif; }
+$font-general: 1.2rem;
 
-h1
-{
+* {
+	font-family: 'Quattrocento Sans', sans-serif;
```

Comment:
The indentation here should use our standard of 4 spaces, not a tab character

Reply:
Not sure why the file is set up like that, my defaults are 4 spaces. Fixing